.gitignore

```diff
@@ -32,10 +32,16 @@ installer/x86_64/platforms/
 src/sonic-config-engine/**/*.pyc
 src/sonic-config-engine/build
 src/sonic-config-engine/sonic_config_engine.egg-info
+
 src/sonic-daemon-base/**/*.pyc
 src/sonic-daemon-base/build
 src/sonic-daemon-base/sonic_daemon_base.egg-info
 
+src/sonic-py-common/**/*.pyc
```

Comment:
>**/*.pyc [](start = 20, length = 8)

pyc should always be ignored, so suggest consolidate them into one line.

Reply:
I think this makes sense in a separate PR designed for general cleanup purposes, as it touches more than this new package.